src/@next/Icon/icons/icons.ts

```diff
@@ -46,6 +46,7 @@ export const iconNames = [
   'ri-close-circle-fill',
   'ri-close-circle-line',
   'ri-close',
+  'ri-coin-line',
```

Comment:
based on the icon, i think should be instead since ri-coin is different?
```suggestion
  'ri-coins-line',
```

Reply:
yep it should be coins, let me fix it up

Reply:
fixed